src/IgnoresAccessChecksToGenerator.Tasks/PublicizeInternals.cs

```diff
@@ -116,12 +118,14 @@ public IgnoresAccessChecksToAttribute(string assemblyName)
 
         private void CreatePublicAssembly(string source, string target)
         {
+            var types = ExcludeTypeNames.Split(Semicolon);
```

Comment:
You didn't mark the property as required so you probably should be checking for null (instead of crashing)